Assets/Scripts/HealthBar.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class HealthBar : MonoBehaviour
+{
+    [SerializeField]
+    private Image foregroundImage;
+
+    public void SetMaxHealth(float percent)
```

Comment:
Same here. Use mutators.
Also `SetHealth` and `SetMaxHealth` are doing the same thing